### ISSUE TITLE
[RFC] merge UI from @hoodie/store & @hoodie/account into hoodie

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/discussion#104]
While working on the new Hoodie Architecture, we created the [`@hoodie/store`](https://github.com/hoodiehq/hoodie-store#readme) and [`@hoodie/account`](https://github.com/hoodiehq/hoodie-account#readme) modules as a way to combine the client & server logic of the store & account core modules, mostly for testing before the whole architecture was ready.

The idea was to make an architecture like this:

```
hoodie
├─┬ @hoodie/account
│ ├── @hoodie/account-client
│ └── @hoodie/account-server
└─┬ @hoodie/store
  ├── @hoodie/store-client
  └── @hoodie/store-server
```

But instead it turned out to more like

```
hoodie
├─┬ @hoodie/client
│ ├── @hoodie/account-client
│ └── @hoodie/store-client
└─┬ @hoodie/server
  ├── @hoodie/account-server
  └── @hoodie/store-server
```

`hoodie` still has [`@hoodie/account` and `@hoodie/store` in its dependencies](https://github.com/hoodiehq/hoodie/blob/45baeaf4023516154ea6b63e83774a77af1c5fc4/package.json#L14-L18). But the only thing we use it for in production is to display the custom UI at `/hoodie/store` ([source](https://github.com/hoodiehq/hoodie-store/tree/master/public)) and `/hoodie/account` ([source](https://github.com/hoodiehq/hoodie-account/tree/master/public)), so just static assets.

We also use it for [account server/client integration tests](https://github.com/hoodiehq/hoodie-account/blob/master/test/integration.js) and  [store server/client integration tests](https://github.com/hoodiehq/hoodie-store/blob/master/test/integration.js)

I would suggest to move both the custom UIs and the tests into the [`hoodie` main repository](https://github.com/hoodiehq/hoodie).

For one, there is no use for the `@hoodie/account` or `@hoodie/store` modules outside Hoodie. And even if there were, it would be simple to create. `@hoodie/client` and `@hoodie/server` on the other side are used as direct dependencies in related projects.

Second, we want to have a custom hoodie UI that every hoodie app gets by default (related: https://github.com/hoodiehq/camp/issues/20, https://github.com/hoodiehq/camp/issues/21, https://github.com/hoodiehq/camp/issues/22). The UI is interdependent though, having the store page come from the store module and the account page from the account module makes little sense. So just having one integrated UI that simply lives at `/hoodie` is much simpler to create and to maintain. And we can use for integration tests of all the core features without needing an app.

tl;dr: this will simplify our architecture, reduce maintenance and it’s fairly simply todo without breaking changes

Let me know what you think @hoodiehq/maintainers @mbad0la @distalx @kouryuu
I suggest to leave this open for discussion until end of day Tuesday, Nov 2nd

**Comment by distalx** [comment]
Totally, sounds good to me!

**Comment by machikoyasuda** [comment]
the folder structure there is already 70% leas confusing! excited for this change

**Comment by rogeriochaves** [comment]
sounds very good to me too 👍

**Comment by gr2m** [comment]
Thanks all! I created an issue at https://github.com/hoodiehq/camp/issues/98
